source/MaterialXView/Main.cpp

```diff
@@ -9,6 +9,10 @@ NANOGUI_FORCE_DISCRETE_GPU();
 const std::string options = 
 " Options: \n"
 "    --material [FILENAME]          Specify the filename of the MTLX document to be displayed in the viewer\n"
+"    --bakeWidth [INTEGER]          Specify the width of the image in pixels (defaults to computed max resolution)\n"
```

Comment:
This documentation could use some clarifications (noting that it's for the width, height, and format of baked textures), and I'd recommend moving these lines just below "captureFilename", which is the most closely-related feature in MaterialXView.